inductiva/tasks/methods.py

```diff
@@ -49,55 +51,49 @@ def _fetch_tasks_from_api(
             raise e
 
 
-def _list_of_tasks_to_str(tasks: Sequence["inductiva.tasks.Task"]) -> str:
+def _dataframe_of_tasks(tasks: Sequence["inductiva.tasks.Task"]) -> str:
     columns = [
-        "ID", "Simulator", "Status", "Submitted", "Started", "Duration",
-        "VM Type"
+        "ID", "Simulator", "Status", "Submitted", "Started", "Computation Time",
+        "Resource Type"
     ]
     rows = []
 
     for task in tasks:
         info = task.get_info()
-        # e.g., get "openfoam" from "fvm.openfoam.run_simulation"
         simulator = task.get_simulator_name()
+        status = task.get_status()
 
-        end_time = info.get("end_time", None)
-        execution_time = task.get_execution_time(fail_if_running=False)
+        computation_end_time = info.get("computation_end_time", None)
 
+        execution_time = task.get_computation_time(fail_if_running=False)
         if execution_time is not None:
-            execution_time = format_utils.seconds_formatter(execution_time)
-            if end_time is None:
-                execution_time = f"*{execution_time}"
+            if computation_end_time is None:
+                if status in ["started", "submitted"]:
+                    execution_time = f"*{execution_time}"
+                else:
+                    execution_time = "n/a"
+
+        executer = info["executer"]
+        if executer is None:
+            resource_type = None
+        else:
+            resource_type = executer["vm_type"]
+            if executer["n_mpi_hosts"] > 1:
+                resource_type += f" x{executer['n_mpi_hosts']}"
 
         row = [
             task.id,
             simulator,
-            task.get_status(),
-            info.get("input_submit_time", None),
-            info.get("start_time", None),
+            status,
+            format_utils.datetime_formatter(info.get("input_submit_time",
+                                                     None)),
```

Comment:
unnecessary line break. It makes it difficult to read.

Reply:
This yapf hitting the boundary 🗡️